<!-- Launch file for the assigner -->
<launch>
  <arg name="number_robots" default="3"/>
  <node pkg="frontier_exploration" type="frontier_filter.py" name="frontier_filter" output="screen">
  <param name="rate" value="2"/>
  <param name="map_topic" value="/map"/>
  <param name="n_robots" value="$(number_robots)"/>
  <param name="distance_threshold" value="3.0"/>
  </node>

</launch>

